modules/example/src/smithy4s/example/package.scala

```diff
@@ -16,6 +16,7 @@ package object example {
   val DeprecatedService = DeprecatedServiceGen
 
   type StreamedBlob = smithy4s.example.StreamedBlob.Type
+  /** This is a simple example of a "quoted string" */
```

Comment:
We should have an example (used in the compilation pipeline, like these files) that contains the string `*/`. Ideally it would be escaped somehow, but I don't have a concrete proposal for _how_ atm.

Reply:
random idea to just do `\*\/`.

Reply:
I was thinking about how to escape `*/` as well, but I didn't get any genius ideas so far. is it just going to be a `docString.replace("*/", "\*\/")`? 

Tho for now I'll focus on getting multi-line to work

Reply:
that `replace` idea sounds good yeah